### PR TITLE
Create documentation-links.yaml

### PR DESCRIPTION
**Pull request by astrojuanlu**
See https://github.com/readthedocs/readthedocs-preview

cc @humitos

### CODE REVIEW

**Comment by astrojuanlu** [comment]
Let's merge and see if it works 😏

File: .github/workflows/documentation-links.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+permissions:
+  pull-requests: write
+
+jobs:
+  documentation-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: readthedocs/readthedocs-preview@main
+        with:
+          project-slug: "readthedocs-preview"
```
Comment (humitos) [inline .github/workflows/documentation-links.yaml:17]: project-slug: "poliastro"

File: .github/workflows/documentation-links.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: Read the Docs Pull Request Preview
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
```
Comment (humitos) [inline .github/workflows/documentation-links.yaml:6]: This `synchronize` is not required.

**Review by humitos — COMMENTED** [review]
I think it should work now. I will try to release a `v1` early so you can pin it and do not have breaking changes.

Take into account this is in beta state, tho 😄

File: .github/workflows/documentation-links.yaml (hunk 1 above)
Comment (astrojuanlu) [reply .github/workflows/documentation-links.yaml:17]: Done ✔️

File: .github/workflows/documentation-links.yaml (hunk 2 above)
Comment (astrojuanlu) [reply .github/workflows/documentation-links.yaml:6]: Done ✔️

**Pull request merged**